.github/workflows/nightly-test-integrations-1.19.x.yml

```diff
@@ -109,7 +109,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        envoy-version: ["1.26.8", "1.27.6", "1.28.5", "1.29.5"]
+        envoy-version: ["1.26.8", "1.27.6", "1.28.4", "1.29.5"]
```

Comment:
This one should fix nightly failures for 1.19.x: https://github.com/hashicorp/consul/actions/runs/9458118650